TTS/utils/audio.py

```diff
@@ -32,6 +32,9 @@ def __init__(
         use_mel=False,
         do_amp_to_db=False,
         spec_gain=1.0,
+        power=None,
+        use_htk=False,
+        mel_norm="slaney"
```

Comment:
Docstring for these ?

What is htk ?

Reply:
All are Librosa parameters. This class doesn't have a docstring, so I need to create it?

Reply:
I added a simple docstring, feel free to improve it.